src/main/io/gps.h

```diff
@@ -35,6 +35,35 @@
 #define GPS_Y 0
 #define GPS_MIN_SAT_COUNT 4      // number of sats to trigger low sat count sanity check
 
+#ifdef USE_GPS_UBLOX
+typedef enum {
+    UBX_VERSION_UNDEF = 0,
+    UBX_VERSION_M5,
+    UBX_VERSION_M6,
+    UBX_VERSION_M7,
+    UBX_VERSION_M8,
+    UBX_VERSION_M9,
+    UBX_VERSION_M10,
+    UBX_VERSION_COUNT,
+} ubloxVersion_e;
+
+extern struct ubloxVersion_t {
+    uint32_t hw;
+    const char* str;
+} ubloxVersion_map[];
+#endif
```

Comment:
```suggestion
struct ubloxVersion_s {
    uint32_t hw;
    const char* str;
};
extern struct ubloxVersion_s ubloxVersion_map[];
#endif
```
This is public interface ...